presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -57,9 +68,25 @@ public void testAggregations()
     }
 
     @Test
-    public void testJoins()
+    public void testJoin()
+    {
+        assertQuery("SELECT * FROM orders o, lineitem l WHERE o.orderkey = l.orderkey AND o.orderkey % 2 = 1");
+    }
+
+    @Test
+    public void testShuffleStats()
     {
         assertQuery("SELECT * FROM orders o, lineitem l WHERE o.orderkey = l.orderkey AND o.orderkey % 2 = 1");
+        Option<AccumulatorV2<?, ?>> accumulator = AccumulatorContext$.MODULE$.lookForAccumulatorByName(PRESTO_SPARK_SHUFFLE_STATS_GENERIC_COLLECTOR);
+        assertTrue(accumulator.isDefined());
+        java.util.List<List<Map<String, Long>>> stats = ((CollectionAccumulator<List<Map<String, Long>>>) accumulator.get()).value();
+        List<Map<String, Long>> metrics = stats.get(0);
+        assertTrue(!metrics.isEmpty());
+        for (Map.Entry<String, Long> entry : metrics.get(0).entrySet()) {
+            String[] desc = entry.getKey().split("\\|");
+            assertTrue(desc.length == 4);
```

Comment:
`assertEqual(4, desc.length)`

  If the test case failed, the `assertTrue` only tells you the check failed but `assertEqual` will tell the value of the `desc.length` which is easier to debug.